src/makeCalculator.js

```diff
@@ -37,7 +37,38 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    add(elem) {
+      calculator.result += elem;
+    },
+
+    subtract(elem) {
+      calculator.result -= elem;
+    },
+
+    multiply(elem) {
+      calculator.result *= elem;
+    },
+
+    divide(elem) {
+      calculator.result /= elem;
+    },
+
+    operate(callback, number) {
+      callback(number);
+
+      return this;
+    },
+    result: 0,
```

Comment:
add result to the start of the object, it's difficult to understand now, what the result on all these methods